chatops/python/docbuilder.py

```diff
@@ -234,7 +234,7 @@ def main():
             except OSError as exception:
                 print_exit('[-] ERR: {0}'.format(exception.strerror),
                            exception.errno)
-        if options['execsummary'] == 'true':  # we're generating a summary as well as a report
+        if options['execsummary'] == True:  # we're generating a summary as well as a report
```

Comment:
In the future, we can refactor this, as execsummary is an 'add-on': first build the execsummary, and don't use the `else` statement for building the report itself.